src/OrchardCore.Build/Dependencies.AspNetCore.props

```diff
@@ -9,7 +9,7 @@
 
     <!-- TFMs used to build the abstractions and modules, by convention the default TFM is at the first position -->
     <!-- In a cross-targeting build, some assets are only copied on the first TFM, by convention the default TFM -->
-    <CommonTargetFrameworks Condition="'$(CommonTargetFrameworks)' == ''">net8.0</CommonTargetFrameworks>
+    <CommonTargetFrameworks Condition="'$(CommonTargetFrameworks)' == ''">net8.0;net7.0;net6.0</CommonTargetFrameworks>
```

Comment:
Did we bring the previous versions back?

Reply:
No ;) This for testing race conditions

Reply:
I see, but why the previous TFM are there?

Reply:
Because race conditions happen when building multiple TFMs at the same time